js/index.js

```diff
@@ -1 +1,6 @@
 export default function igArticle(){};
+
+igArticle.init = function() {
+  window.isNext = document.cookie.replace(/(?:(?:^|.*;\s*)FT_SITE\s*\=\s*([^;]*).*$)|^.*$/, "$1");
```

Comment:
@kavanagh I've created a new utility class, `.wait-for-js`, that sets elements with it to `display: none`. The `igArticle.init` method then removes them all. This should prevent that.